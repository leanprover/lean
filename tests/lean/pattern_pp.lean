definition Sum : nat → (nat → nat) → nat := sorry
attribute [forward]
definition Sum_weird (f g h : nat → nat) (n : nat) : (Sum n (λ x, f (g (h x)))) = 1 := sorry
print Sum_weird

/-
attribute [forward]
definition Sum_weird : ∀ (f g h : nat → nat) (n : nat), eq (Sum n (λ (x : nat), f (g (h x)))) 1 :=
λ (f g h : nat → nat) (n : nat), sorry
(multi-)patterns:
?M_1 : nat → nat, ?M_2 : nat → nat, ?M_3 : nat → nat, ?M_4 : nat
{Sum ?M_4 (λ (x : nat), ?M_1)}
-/
