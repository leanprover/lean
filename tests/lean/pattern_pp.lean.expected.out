pattern_pp.lean:1:11:failed to generate bytecode for 'Sum'
code generation failed, VM does not have code for 'sorry'
attribute [forward]
definition Sum_weird : ∀ (f g h : ℕ → ℕ) (n : ℕ), Sum n (λ (x : ℕ), f (g (h x))) = 1 :=
λ f g h n, sorry
